tasks/eslint.js

```diff
@@ -5,14 +5,21 @@
 module.exports = function (grunt) {
   grunt.config('eslint', {
     options: {
-      eslintrc: '.eslintrc'
-    },
-    app: {
+      eslintrc: '.eslintrc',
       src: ['Gruntfile.js', 'tasks/*.js', 'config/**/*.js', 'node/**/*.js']
     },
-    client: {
+    app: {
```

Comment:
I think something weird happened here. the original `app` target's `src` properties got merged up into `options`, and the `client` task was renamed to `app`.